### PR TITLE
Fix exercism non-unique test names

### PR DESCRIPTION
**Pull request by mpizenberg**

### CODE REVIEW

**Comment by mpizenberg** [comment]
@jiegillet and @ceddlyburge here is the fix for the non-unique test names issue you discovered.

```elm
{-| Use all description levels, except the first one (exercise name), to build the test name.
-}
extractTestName : List String -> String
extractTestName labels =
    List.reverse labels
        |> List.drop 1
        |> String.join " > "
```

Once I merge it and merge #10 I'll publish a new version of this elm package. Then publish a new version of elm-test-rs.

We won't be able to update the `elm-tooling.json` of the track repo because it will need an update of both elm-tooling and the elm-tooling action. It's not a big issue since the CI tests there do not merge identical tests names.

We will be able to update the version of elm-test-rs in the docker for exercism test runner since it's just a download from GitHub releases.

**Review by ceddlyburge — APPROVED** [review]

**Review by jiegillet — APPROVED** [review]
Nice!
I did the equivalent in exercism/elm-test-runner so that the names can match [here](https://github.com/exercism/elm-test-runner/pull/38).

**Pull request merged**
